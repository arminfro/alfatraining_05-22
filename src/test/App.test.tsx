import { matchSnapshot, renderWithRouting, screen } from "./utils";

const renderApp = () => renderWithRouting();

test("matches snapshot", () => matchSnapshot(renderApp()));

test("renders: Willkommen beim BookMonkey from Home Component", async () => {
  renderApp();
  const textElement = await screen.findByText(/Willkommen beim BookMonkey/i);
  expect(textElement).toBeInTheDocument();
});
