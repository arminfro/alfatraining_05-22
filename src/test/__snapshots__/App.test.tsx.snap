// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<body>
  <div>
    <h4
      class="title"
    >
      Willkommen beim BookMonkey
    </h4>
    <div
      class="section"
    >
      <a
        class="button is-link mt-2"
        href="/books"
      >
        Buchliste ansehen
      </a>
    </div>
    <div
      class="section"
    >
      <div
        style="width: 800px;"
      >
        <h2
          class="title"
        >
          Book Search
        </h2>
        <div
          class="panel-block"
        >
          <p
            class="control has-icons-right"
          >
            <input
              class="input"
              type="text"
              value=""
            />
            <span
              class="icon is-small is-right"
            >
              <i
                class="ion-search-outline"
              />
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</body>
`;
